src/components/People/People.tsx

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable no-nested-ternary */
+import { useEffect, useState } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+
+export const People = () => {
+  const { pathname } = useLocation<{ pathname: string }>();
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [slugUser, setSlugUser] = useState(pathname.slice(8));
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    const fetchPeople = async () => {
```

Comment:
We already have this function to get people so please try to use it, dont need to declared new one, `getPeople()` look` import { getPeople } from '../api';`

Reply:
I use getPeople() in code below and just give some new things(setIsLoading, catch error)
useEffect(() => {
    const fetchPeople = async () => {
      setIsLoading(true);
      try {
        const data = await getPeople();

        setPeople(data);
      } catch (error) {
        setErrorMessage('Unable to load people');
      }

      setIsLoading(false);
    };

    fetchPeople();
  }, []);